test/python/test_python_algorithm.py

```diff
@@ -0,0 +1,116 @@
+from teca import *
```

Comment:
We should avoid wildcard imports, especially in example code, again following PEP 8: https://pep8.org/#imports

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools.